Rocket.Chat/Controllers/Subscriptions/SubscriptionsList/SubscriptionsViewModel.swift

```diff
@@ -47,18 +48,30 @@ class SubscriptionsViewModel {
         }
     }
 
+    var reloadNotificationToken: NotificationToken?
     let realm: Realm?
 
     init(realm: Realm? = Realm.current) {
         self.realm = realm
+        observeAuth()
     }
 
     deinit {
         assorter?.invalidate()
     }
 
+    func observeAuth() {
+        reloadNotificationToken = realm?.objects(Auth.self).observe({ [weak self] _ in
+            if self?.realm?.objects(Auth.self).count == 1 {
+                DispatchQueue.main.async {
+                    self?.reloadData?()
```

Comment:
@Sameesunkaria this is to make sure we will have the Auth object accessible when first loading the `SubscriptionsViewController`?

Reply:
This is to reload the cells so that the avatar views get updated when the server url is accessible. This is required after the first login. If you have a better solution, please let me know.

Reply:
The solutions looks good, I was just thinking that we could invalidate the token after executing the block once. It seems that it only needs to be executed once. Does it makes sense?

Reply:
It gets called a few times, so if we invalidate it before that (after the first time the block executes) it sometimes causes the images to not be loaded. I am not sure as to why it gets called more than once. If you can figure that out, maybe we'll be able to invalidate that notification.

Reply:
Got it. Let's leave it this way, it's not critical, we can improve in another PR. Thanks for explaining